src/assets/data/markdown/ionic-weather-step07-01.md

```diff
@@ -30,28 +26,6 @@ import { SplashScreen } from '@ionic-native/splash-screen';
 import { IconMapProvider } from '../providers/icon-map/icon-map';
 import { WeatherProvider } from '../providers/weather/weather';
 
```

Comment:
Same question here... I thought there was a lab earlier where we added this stuff. I don't remember removing that lab. Perhaps we should.